src/backend/distributed/test/metadata_sync.c

```diff
@@ -14,6 +14,7 @@
 #include "fmgr.h"
 
 #include "catalog/pg_type.h"
+#include "distributed/listutils.h"
 #include "distributed/metadata_sync.h"
 #include "distributed/test_helper_functions.h" /* IWYU pragma: keep */
```

Comment:
Well, this was messy.

Maybe remove some of the `#include "distributed/test_helper_functions.h"` lines.